messaging/Controllers/BundlesController.cs

```diff
@@ -29,19 +29,39 @@ public BundlesController(ILogger<BundlesController> logger, ApplicationDbContext
             Services = services;
             _settings = settings.Value;
             _logger = logger;
+
         }
 
         // GET: Bundles
         [HttpGet]
-        public async Task<ActionResult<Bundle>> GetOutgoingMessageItems(string jurisdictionId, DateTime _since = default(DateTime))
+        public async Task<ActionResult<Bundle>> GetOutgoingMessageItems(string jurisdictionId, int _count, DateTime _since = default(DateTime), int page = 1)
         {
+            if (_count == 0) 
+            {
+                _count = _settings.PageCount;
+            }
+
             if (!VRDR.MortalityData.Instance.JurisdictionCodes.ContainsKey(jurisdictionId))
             {
                 // Don't log the jurisdictionId value itself, since it is (known-invalid) user input
                 _logger.LogError("Rejecting request with invalid jurisdiction ID.");
                 return BadRequest();
             }
 
+            if (_count < 0)
+            {
+                return BadRequest("_count must not be negative");
+            }
+            if (page < 1)
+            {
+                return BadRequest("page must not be negative");
+            }
+            // Retrieving unread messages changes the result set (as they get marked read), so we don't REALLY support paging
+            if (_since == default(DateTime) && page > 1)
+            {
+                return BadRequest("Paging not supported");

```

Comment:
I think the message here could be clearer, maybe something like "Pagination does not support specifying a page other than the immediate next available page"